### ISSUE TITLE
how to configure from e-mail address and leave msg form

### ISSUE THREAD

**Issue by gituser** [LiveHelperChat/livehelperchat#227]
Hi.

We've successfully installed LiveHelperChat on sub-domain.

1) we've used a widget with specific settings: 

when there is no operator online it asks for a name, e-mail and message and passes e-mail to the operator.

if user sends a message through the widget on the site -- operator receives the e-mail just fine with back-address of what user typed into e-mail field, so operator can just hit reply and reply to the user request via e-mail.

but if user goes directly to http://livechat.domain.com/ he sees only 1 field (message) and if he send something there -- chat opens and waiting for an operator.

i've checked setttings -- there is a 'chat configuration options' you can ofc require e-mail, name by default but it's not really convinient, because its only needed only if operator is offline otherwise no need to require e-mail address.

2) 2nd issue is from address.

if user sends through the widget when operator offline - email comes fine and has backward address of what user typed

but if user goes from the frontpage of http://livechat.domain.com/ (i set to require e-mail always there) -- operator receives message from root@localhost and it quickly goes to the spam.

i've checked the source and noticed there is phpmailer used but couldn't find how to fix this behavior.

To summarize what I propose: 

1) add settings for online / offline operator mode separately to control what data user should input globally (e.g. for http://livechat.domain.com/ -- chat configuration).

2) add a setting to specify backward e-mail address (default address, e.g. livechat@domain.com instead of dodgy root@localhost).

many thanks!
